elixir/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.10.4-alpine
+FROM hexpm/elixir:1.14.3-erlang-25.2.3-alpine-3.17.0
```

Comment:
can you keep this at 3.16 for now? I tried to enforce the same alpine version where possible

Reply:
3.16.4?